string NAME=collided

ObjectLocation l0
ObjectLocation l1

bool truth_value
